test/test_site/index.md

```diff
@@ -68,3 +68,13 @@ siteNav: site-nav.md
 <panel header="## Panel with src from another Markbind site header" src="sub_site/index.md" expanded>
 </panel>
 </div>
+
+# modal with panel inside
+<trigger for="modal-with-panel">trigger</trigger>
+
+<modal title="modal with panel inside" id="modal-with-panel">
```

Comment:
As discussed, need to differentiate between the headings of line 72 and 75 (should give different names for each, otherwise it is hard to tell which one is being indexed).